core/email_api.php

```diff
@@ -1300,6 +1300,16 @@ function email_send( EmailData $p_email_data ) {
 			break;
 	}
 
+	#apply DKIM settings
+	if( config_get( 'dkim_enable' ) ) {
+		$t_mail->DKIM_domain = config_get( 'email_dkim_domain' );
+		$t_mail->DKIM_private = config_get( 'email_dkim_private_key' );
```

Comment:
`private_key` is certainly better than `private`, but wouldn't something like `private_key_file_path` be even better?